src/components/stegvisning/AFP/AFP.tsx

```diff
@@ -104,56 +114,86 @@ export function AFP({
         <BodyLong size="large">
           <FormattedMessage id="stegvisning.afp.ingress" />
         </BodyLong>
-        <ReadMore
-          name="Avtalefestet pensjon i offentlig sektor"
-          className={styles.readmoreOffentlig}
-          header={
-            <FormattedMessage id="stegvisning.afp.readmore_offentlig_title" />
-          }
-        >
-          <FormattedMessage id="stegvisning.afp.readmore_offentlig_list_title" />
-          <ul className={styles.list}>
-            <li>
-              <FormattedMessage id="stegvisning.afp.readmore_offentlig_list_item1" />
-            </li>
-            <li>
-              <FormattedMessage id="stegvisning.afp.readmore_offentlig_list_item2" />
-            </li>
-            <li>
-              <FormattedMessage id="stegvisning.afp.readmore_offentlig_list_item3" />
-            </li>
-          </ul>
-          <FormattedMessage id="stegvisning.afp.readmore_offentlig_ingress" />
-        </ReadMore>
-        <ReadMore
-          name="Avtalefestet pensjon i privat sektor"
-          className={styles.readmorePrivat}
-          header={
-            <FormattedMessage id="stegvisning.afp.readmore_privat_title" />
-          }
-        >
-          <FormattedMessage id="stegvisning.afp.readmore_privat_list_title" />
-          <ul className={styles.list}>
-            <li>
-              <FormattedMessage id="stegvisning.afp.readmore_privat_list_item1" />
-            </li>
-            <li>
-              <FormattedMessage id="stegvisning.afp.readmore_privat_list_item2" />
-            </li>
-            <li>
-              <FormattedMessage id="stegvisning.afp.readmore_privat_list_item3" />
-            </li>
-            <li>
-              <FormattedMessage id="stegvisning.afp.readmore_privat_list_item4" />
-            </li>
-          </ul>
-          <FormattedMessage
-            id="stegvisning.afp.readmore_privat_link"
-            values={{
-              ...getFormatMessageValues(),
-            }}
-          />
-        </ReadMore>
+
+        {sanityFeatureToggle?.enabled && readMore1 ? (
+          <ReadMore
+            data-testid={readMore1.name}
+            name={readMore1.name}
+            header={readMore1.overskrift}
+            className={styles.readmoreOffentlig}
+          >
+            <PortableText
+              value={readMore1.innhold}
+              components={getSanityPortableTextComponents(intl)}
```

Comment:
Jeg ser at den brukes med spread operator i Utenlandsopphold og ikke her. Hvis ikke det er behov for spread kan den gjerne fjernes fra Utenlandsopphold slik at det blir likt :)